src/MergeContext.js

```diff
@@ -108,6 +108,82 @@ class Approval {
     }
 }
 
+class StatusCheck
+{
+    constructor(raw) {
+        assert(raw.context);
+        assert(raw.state);
+        assert(raw.target_url);
+        assert(raw.description);
+
+        this.context = raw.context;
+        this.state = raw.state;
+        this.targetUrl = raw.target_url;
+        this.description = raw.description;
+
+        this._raw = raw;
+    }
+}
+
+// Passed required status checks analysis
+class StatusResult
```

Comment:
s/StatusResult/StatusChecks/gi and describe the class as `aggregates (required) status check for a PR or commit`